build/gulpfile.sql.js

```diff
@@ -113,25 +114,23 @@ gulp.task('install-sqltoolsservice', () => {
 	return installService();
 });
 
-function installSsmsMin() {
+gulp.task('install-ssmsmin', () => {
```

Comment:
I think there are hooks we could move this too for the vsix utility.

Reply:
Looks like there are - https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prepublish-step

Could also add compiling as part of that to combine everything into one step

Reply:
We run the "webpack" steps which moves the files before running the vsix creator, so "dev" files like compiling and downloading the ssmin script would not be present when running the vsix creator. We could modify their webpack stuff to follow the vsce pre:publish steps.